mk_verificator/tests/test_floating_ip.py

```diff
@@ -0,0 +1,19 @@
+import mk_verificator.utils.ssh_client as ssh
+import mk_verificator.clients.nova as nova
+
+
+def test_floating_ip():
```

Comment:
please, use setup/teardown fixtures because of for now we will not delete VM after `assert` statement if it's failed